GVRf/Framework/framework/src/main/java/org/gearvrf/GVRSceneObject.java

```diff
@@ -639,19 +671,23 @@ public void detachEyePointeeHolder() {
      * @return The {@link GVREyePointeeHolder} attached to the object. If no
      *         {@link GVREyePointeeHolder} is currently attached, returns
      *         {@code null}.
+     * @deprecated use getCollider or getComponent(GVRCollider.getComponenType())
      */
     public GVREyePointeeHolder getEyePointeeHolder() {
-        return (GVREyePointeeHolder) getComponent(GVREyePointeeHolder.getComponentType());
+        GVRCollider collider = getCollider();
+        if ((collider != null) && GVREyePointeeHolder.class.isAssignableFrom(collider.getClass())) {
+            return (GVREyePointeeHolder) collider;
+        }
+        return null;
     }
 
     /**
      * Simple, high-level API to enable or disable eye picking for this scene
      * object.
      * 
-     * The {@linkplain #attachEyePointeeHolder(GVREyePointeeHolder) low-level
+     * The {@linkplain #attachCollider low-level
      * API} gives you a lot of control over eye picking, but it does involve an
-     * awful lot of details. Since most apps are just going to use the
-     * {@linkplain #attachEyePointeeHolder() simple API} anyhow, this method
+     * awful lot of details. This method
```

Comment:
Should mention that using this method would attach a sphere collider as default rather than a mesh collider.